cpp/ql/src/Security/CWE/CWE-704/WcharCharConversion.ql

```diff
@@ -0,0 +1,29 @@
+/**
+ * @name Cast between semantically different string types: char* from/to wchar_t*
```

Comment:
I'd like a shorter `@name` for consistency with our existing queries. I propose "Cast from char* to wchar_t*".